### PR TITLE
Lowercase the Block Editor and Classic Editor strings 

### PR DESCRIPTION
**Pull request by garrett-eclipse**
To match the core convention for lowercasing the block editor and classic editor strings unless it's a title or referring to Classic Editor plugin
Reference - https://make.wordpress.org/core/handbook/best-practices/spelling/

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by kristastevens** [comment]
@yaelirub, thanks for the ping! Is there specific text you'd like us to look at? If this is more a philosophical question about approach, then yes, we'd agree that consistency with the Make WordPress Core Handbook is the way to go.

**Comment by yaelirub** [comment]
Thanks for looking into this, @kristastevens .
Yes, it's more of a philosophical question :)

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift [hunk 1]
```diff
@@ -15,10 +15,10 @@ extension GutenbergViewController {
     enum InfoDialog {
         static let key = "Gutenberg.InformativeDialog"
         static let message = NSLocalizedString(
-            "This post was originally created in the Block Editor, so we've also enabled it on this Post. Switch back to Classic at any time by tapping ••• in the top bar.",
-            comment: "Popup content about why this post is being opened in Block Editor"
+            "This post was originally created in the block editor, so we've also enabled it on this Post. Switch back to the classic editor at any time by tapping ••• in the top bar.",
+            comment: "Popup content about why this post is being opened in block editor"
         )
-        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in Block Editor")
+        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in block editor")
```
Comment (yaelirub) [inline WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift]: Should this be "Block editor enabled"?

Comment (kristastevens) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift]: I think so, yes. We use sentence case as a default.

Comment (garrett-eclipse) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift]: Thanks @yaelirub & @kristastevens updating coming in

**Comment by garrett-eclipse** [comment]
Thanks @yaelirub I've updated that one string

**Comment by garrett-eclipse** [comment]
Latest commit to branch is to resync with upstream/develop

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift [hunk 2]
```diff
@@ -15,7 +15,7 @@ extension GutenbergViewController {
         /*if mode == .richText {
             // NB : This is a candidate for plurality via .stringsdict, but is limited by https://github.com/wordpress-mobile/WordPress-iOS/issues/6327
             let textCounterTitle = String(format: NSLocalizedString("%li words, %li characters", comment: "Displays the number of words and characters in text"), richTextView.wordCount, richTextView.characterCount)
-
```
Comment (jkmassel) [inline WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift]: Can we remove this whitespace change?

Comment (garrett-eclipse) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift]: I can't seem to whenever I introduce and save it auto-strips. Should it even be there though as no other empty lines have white space at the beginning like that

**Comment by garrett-eclipse** [comment]
> Can we please remove changes to `WordPress/Resources/AppStoreStrings.po` before merging?

Sorry about that tried to sync with develop and resolve conflicts didn't go well so I rebased and removed that commit.

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift (hunk 2 above)
Comment (jkmassel) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift]: Hmmm, good point. It's not ideal to have this in an unrelated PR, but it's even less ideal to have it in the code in the first place. I'm ok to remove it!

**Comment by jkmassel** [comment]
@etoledom – I merged this branch with `develop` – as of https://github.com/wordpress-mobile/WordPress-iOS/pull/11921/commits/1c30b4e0439f8748ed06fe17d8ec4be70b192c23, this branch should align with https://github.com/wordpress-mobile/WordPress-iOS/blob/dbe265afa5528c918270785326737805499a825c/WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController%2BInformativeDialog.swift#L8-L11

Looking for your 👍 on that prior to merge, if it's not too much trouble? :)

**Comment by jkmassel** [comment]
I fixed up a merge issue, but unfortunately, Hound is unhappy now, which prevents us from merging. To resolve the line length issue, I'd suggest using the syntax from https://github.com/wordpress-mobile/WordPress-iOS/blob/dbe265afa5528c918270785326737805499a825c/WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController%2BInformativeDialog.swift#L8-L11

Once that's fixed, and we get the 👍 from @etoledom, I'm good to :shipit:

**Comment by garrett-eclipse** [comment]
Hi @jkmassel is that 'To resolve the line length issue, I'd suggest using the syntax from' item addressed to me or @etoledom let me know if anything is required from me and I can make an update

**Comment by jkmassel** [comment]
Ah sorry for the confusion! If you're able to implement the new syntax, that would be fantastic (and Hound will really appreciate it too!)

**Comment by garrett-eclipse** [comment]
k @jkmassel I'll take a look over the weekend here, gotta run and get the kids

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift [hunk 3]
```diff
@@ -9,7 +9,7 @@ extension GutenbergViewController {
             "This post uses the block editor, which is the default editor for new posts. To enable the classic editor, go to Me > App Settings.",
             comment: "Popup content about why this post is being opened in block editor"
         )
-        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in Block Editor")
+        static let title = NSLocalizedString("Block editor enabled", comment: "Popup title about why this post is being opened in block editor")
```
[the remark this answers is not in the document]
Comment (garrett-eclipse) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift]: addressed in 6409735

File: WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift [hunk 4]
```diff
@@ -3228,7 +3228,7 @@ extension AztecPostViewController {
     }
 
     struct MoreSheetAlert {
-        static let gutenbergTitle = NSLocalizedString("Switch to Block Editor", comment: "Switches from the Classic Editor to Block Editor.")
+        static let gutenbergTitle = NSLocalizedString("Switch to block editor", comment: "Switches from the classic editor to block editor.")
```
[the remark this answers is not in the document]
Comment (garrett-eclipse) [reply WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift]: addressed in 6409735

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift [hunk 5]
```diff
@@ -94,7 +94,10 @@ extension GutenbergViewController {
 
 extension GutenbergViewController {
     private struct MoreSheetAlert {
-        static let classicTitle = NSLocalizedString("Switch to Classic Editor", comment: "Switches from Gutenberg mobile to the Classic editor")
+        static let classicTitle = NSLocalizedString(
+            "Switch to classic editor",
```
[the remark this answers is not in the document]
Comment (garrett-eclipse) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift]: addressed in 4c8a136

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift [hunk 6]
```diff
@@ -9,7 +9,10 @@ extension GutenbergViewController {
             "This post uses the block editor, which is the default editor for new posts. To enable the classic editor, go to Me > App Settings.",
             comment: "Popup content about why this post is being opened in block editor"
         )
-        static let title = NSLocalizedString("Block Editor Enabled", comment: "Popup title about why this post is being opened in Block Editor")
+        static let title = NSLocalizedString(
+            "Block editor enabled",
```
[the remark this answers is not in the document]
Comment (garrett-eclipse) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift]: addressed in 4c8a136

File: WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift [hunk 7]
```diff
@@ -3228,7 +3228,10 @@ extension AztecPostViewController {
     }
 
     struct MoreSheetAlert {
-        static let gutenbergTitle = NSLocalizedString("Switch to Block Editor", comment: "Switches from the Classic Editor to Block Editor.")
+        static let gutenbergTitle = NSLocalizedString(
+            "Switch to block editor",
```
[the remark this answers is not in the document]
Comment (garrett-eclipse) [reply WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift]: addressed in 4c8a136

**Comment by garrett-eclipse** [comment]
Alright, I think I appeased the beast ;)
"No violations found. Woof!"

Can you take a look @jkmassel 
As to your unresolved change request I was addressed in when I rebased. But because the file isn't part of the PR anymore so the change request conversion is gone and I can't mark resolved. Referring to this;
> Can we please remove changes to WordPress/Resources/AppStoreStrings.po before merging?
* did this by rolling back the commit.

**Comment by jkmassel** [comment]
This is ready to merge. Thanks @garrett-eclipse!!

**Comment by garrett-eclipse** [comment]
Awesome thanks for getting it to the endzone @jkmassel

**Pull request merged**